core/src/main/java/de/danoeh/antennapod/core/feed/FeedUrlNotFoundException.java

```diff
@@ -0,0 +1,19 @@
+package de.danoeh.antennapod.core.feed;
+
+public class FeedUrlNotFoundException extends RuntimeException {
```

Comment:
To me, using a `RuntimeException` for this feels a bit weird. I would just extend `Exception`, or maybe even `IoException`. Could you also please overwrite the `getMessage` function to return something like `"Result does not specify a feed url"`?